docs/Introduction/architecture-overview/architecture-decentralized-model.md

```diff
@@ -26,8 +26,8 @@ Each price feed is updated by multiple, independent Chainlink oracle operators.
   "images": [
     {
       "image": [
-        "https://files.readme.io/bf08b31-5ef7eba-screenshot.png",
-        "5ef7eba-screenshot.png",
+        "https://i.imgur.com/QaJm3i7.png",
```

Comment:
Do we have a policy for where we host images?

Also, the caption on line 35 should be `data.chain.link` instead of `feeds.chain.link`.